src/migrations/20210421133845-add-sort-order-to-strategies.js

```diff
@@ -6,7 +6,6 @@ exports.up = function (db, cb) {
     ALTER TABLE strategies ADD COLUMN sort_order integer DEFAULT 9999;
     UPDATE strategies SET sort_order = 0 WHERE name = 'default';
     UPDATE strategies SET sort_order = 1 WHERE name = 'flexibleRollout';
-    UPDATE strategies SET sort_order = 2 WHERE name = 'userWithId';
```

Comment:
I'm pretty sure we shouldn't update existing migrations and that this would silently fail if it can't find any row with `name = 'userWithId'`.

Reply:
Instead, we should add a new migration that removes the strategy.

Reply:
Yes, it's intentional, that way we only affect new installations and don't change existing installations. We don't want to remove the strategy on existing installations, only if it's new.

It's true that the other migrations would not apply if you don't find the strategy due to the where filter, but it feels cleaner to remove those.